src/SmartComponents/Recs/Details.js

```diff
@@ -218,17 +219,18 @@ const OverviewDetails = ({ match, fetchRuleAck, fetchTopics, fetchSystem, fetchR
                                         className='ins-c-rec-details__actions_dropdown'
                                         onSelect={() => setActionsDropdownOpen(!actionsDropdownOpen)}
                                         position='right'
+                                        ouiaId="actions"
                                         toggle={<DropdownToggle
                                             isDisabled={!permsDisableRec}
                                             onToggle={(actionsDropdownOpen) => setActionsDropdownOpen(actionsDropdownOpen)}
                                             toggleIndicator={CaretDownIcon}>Actions
                                         </DropdownToggle>}
                                         isOpen={actionsDropdownOpen}
                                         dropdownItems={rule && rule.rule_status === 'enabled' ?
-                                            [<DropdownItem key='link'
+                                            [<DropdownItem key='link' ouiaId="item1"
```

Comment:
should these ids be more descriptive? "disable" or "enable"

Reply:
I agree, the rest are fine but these are generic and aren't terribly helpful as is.

Reply:
Agree!
I was just not sure what should be the proper id.
Thanks for reviewing :)